docs/machine-learning/tutorials/tensorflow-transfer-learning.md

```diff
@@ -0,0 +1,337 @@
+
+### Initialize variables in Main
+
+Initialize the `_mlContext` global variable  with a new instance of `MLContext`.  Replace the `Console.WriteLine("Hello World!")` line with the following code in the `Main` method:
+
+[!code-csharp[CreateMLContext](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#CreateMLContext)]
+
+## Load the data
+
+ As the input and output of [`Transforms`](../basic-concepts-model-training-in-mldotnet.md#transformer), a `DataView` is the fundamental data pipeline type, comparable to `IEnumerable` for `LINQ`.
+
+In ML.NET, data is similar to a `SQL view`. It is lazily evaluated, schematized, and heterogenous. The object is the first part of the pipeline, and loads the data.  The `DataView` is used to create and train the model.
+
+Since your previously created `ImageNetData` data model type matches the dataset schema, you can combine the initialization, mapping, and dataset loading into one line of code.
+
+Load the data using the `MLContext.Data.ReadFromTextFile` wrapper. It returns a
+<xref:Microsoft.Data.DataView.IDataView> which infers the dataset schema from the `ImageNetData` data model type.
+
+## Build and Train the model
+
+Add the following call to the `BuildAndTrainModel`method as the next line of code in the `Main` method:
+
+[!code-csharp[CallBuildAndTrainModel](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#CallBuildAndTrainModel)]
+
+The `BuildAndTrainModel` method executes the following tasks:
+
+* Loads the data
+* Extracts and transforms the data.
+* Scores the TensorFlow model.
+* Trains the model.
+* Displays model results.
+* Evaluates the model.
+* Saves the model.
+
+Create the `BuildAndTrainModel` method, just after the `Main` method, using the following code:
+
+```csharp
+public static void BuildAndTrainModel(MLContext mlContext, string dataLocation, string imagesFolder, string inputModelLocation, string outputModelLocation)
+{
+
+}
+```
+
+## Extract Features and transform the data
+
+Pre-processing and cleaning data are important tasks that occur before a dataset is used effectively for machine learning. Raw data is often noisy and unreliable, and may be missing values. Using data without these modeling tasks can produce misleading results.
+
+ML.NET's transform pipelines compose a custom `transforms`set that is applied to your data before training or testing. The transforms' primary purpose is data [featurization](../resources/glossary.md#feature-engineering). Machine learning algorithms understand [featurized](../resources/glossary.md#feature) data, so the next step is to transform our textual data into a format that our ML algorithms recognize. That format is a [numeric vector](../resources/glossary.md#numerical-feature-vector).
```

Comment:
How does this relate to this particular tutorial, Is there textual data for this tutorial?